appdaemontestframework/automation_fixture.py

```diff
@@ -13,7 +13,7 @@ class AutomationFixtureError(AppdaemonTestFrameworkError):
 def _instantiate_and_initialize_automation(function, automation_class, given_that, hass_functions):
     _inject_helpers_and_call_function(function, given_that, hass_functions)
     automation = automation_class(
-        None, None, None, None, None, None, None, None)
+        None, automation_class.__name__, None, None, None, None, None, None)
```

Comment:
Could you please update the doc as well, to reflect that we're setting the name now in `@automation_fixture`.

Here is the relevant section: https://floriankempenich.github.io/Appdaemon-Test-Framework/#automation_fixture

As for the wording: As you wish, and if I feel it's confusing we'll simply discuss about it quickly, no pressure :)

Reply:
sounds good, I took a stab at it.

Reply:
Great! 👍